remind_class.py

```diff
@@ -0,0 +1,77 @@
+import email, smtplib, ssl
+from email.message import EmailMessage
+import time 
+from datetime import datetime
+
+################################################################################
+def look_up_gateway():
+    gate_way = {'AirFire Mobile': "number@sms.airfiremobile.com", 'Aio Wireless': "number@mms.aiowireless.net", 
+    'Alaska Communications': "number@msg.acsalaska.com", 'Alltel (Allied Wireless)' : "number@mms.alltelwireless.com", 'Verizon': "number@vzwpix.com.",
+    "Assurance Wireless": "number@vmobl.com", "AT&T": "number@mms.att.net", "BellSouth": "number@bellsouth.cl", "Bluegrass Cellular": "number@mms.myblueworks.com",
+    'Bluesky Communications': "number@psms.bluesky.as", 'Boost mobile': "number@myboostmobile.com", 'CellCom': "number@cellcom.quiktxt.com", 'Cellular South':"number@csouth1.com",
+    'Centennial Wireless': "number@cwemail.com", 'Chariton Valley Wireless': "number@sms.cvalley.net", 'Chat Mobility': "number@mail.msgsender.com",
+    'Cincinnati Bell': "number@mms.gocbw.com", 'Cleartalk': "number@sms.cleartalk.us", 'Cricket': "number@mms.cricketwireless.net", 'C Spire Wireless': "number@cspire1.com", 
+    'Edge Wireless': "number@sms.edgewireless.com", 'Element Mobile': "number@SMS.elementmobile.net", 'General Communications Inc.': "number@mobile.gci.net", 
+    'Golden State Cellular': "number@gscsms.com", 'GreatCall': "number@vtxt.com", 'MetroPCS': "number@mymetropcs.com", 'Pocket Wireless': "number@sms.pocket.com", 'Red Pocket Mobile':"number@vtext.com"}   
```

Comment:
Even though it will make this file really long, I think it would be better to have one carrier per line.

```
    gate_way = {
        'AirFire Mobile': "number@sms.airfiremobile.com", 
        'Aio Wireless': "number@mms.aiowireless.net", 
        etc
        }
```